simulation-app/database/Dockerfile

```diff
@@ -0,0 +1,5 @@
+FROM postgres:13.3-alpine
+ENV POSTGRES_PASSWORD=postgres
+ENV POSTGRES_USER=postgres
+ENV POSTGRES_DB=atlas_simulation
+COPY init.sql /docker-entrypoint-initdb.d/init.sql
```

Comment:
This is good and exactly what is needed, but I don't think we need to nest it into the `simulation-app/` directory. Instead, you can just move the `database/` directory with its respective files up to the top level directory so that each directory of that "atlast-dashboard" project represents it's own application that runs and talks to the other parts.